Sources/PackageModel/UserToolchain.swift

```diff
@@ -117,15 +117,81 @@ public final class UserToolchain: Toolchain {
         return toolPath
       }
     }
+
+    if useXcrun {
+      #if os(Windows)
+        func getHostTriple(from destination: Destination, swiftCompilerPath: AbsolutePath?) -> Triple? {
+          if let triple = destination.hostTriple {
+            return triple
+          } else if let swiftCompilerPath = swiftCompilerPath {
+            return .getHostTriple(usingSwiftCompiler: swiftCompilerPath)
+          } else {
+            return nil
+          }
+        }
+        if let hostTriple = getHostTriple(from: destination, swiftCompilerPath: swiftCompilerPath),
+           let tool = findToolInVisualStudio(name, triple: destination.targetTriple, hostTriple: hostTriple) {
+          return tool
+        }
+      #endif
+    }
```

Comment:
This makes the behaviour different when targeting Windows from Windows and Windows from another environment just making things more complicated and confusing for users.

Reply:
That's what `useXcrun` is already doing for Darwin to Darwin case, which I believe is reasonable — just like `xcrun` is only available on macOS, Visual Studio is basically a Windows thing.